helm_chart/recordingbot/values.yaml

```diff
@@ -0,0 +1,18 @@
+# global:
+app:
+  replica: 3
+  image: quay.io/asagtani_wire/recordingbot
+  tag: latest
+  env:
+    - name: SERVICE_TOKEN
+      value: # Provide the Recording bot authentication token here
+    - name: PUBLIC_URL
+      value: http://www.example.com # Link to the subdomain pointing to recordingbot
+    - name: DB_URL
+      value: jdbc:postgresql://data-postgres:5432/recording # Link to your Postgres server
+    - name: DB_USER
+      value: recording # Provide your Postgres DB username
+    - name: DB_PASSWORD
+      value: recording # Provide your Postgres DB password
```

Comment:
This will expose passwords directly in the deployment spec. `DB_URL`, `DB_USER` and `DB_PASSWORD` need to be put in a secret, and its values provided as env variables.

I'd probably ask for these inside a `config` helm value dict.

Reply:
In the other wire-server charts we are setting all the secrets in the values.yaml instead of exporting them as env vars, to keep the consistency, moving them to values.yaml and encoding the secrets wherever needed